metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -233,11 +810,13 @@ def clean_host_port(cls, v):
 @support_status(SupportStatus.TESTING)
 @capability(SourceCapability.PLATFORM_INSTANCE, "Enabled by default")
 @capability(SourceCapability.DOMAINS, "Supported via the `domain` config field")
-class VerticaSource(SQLAlchemySource):
+class VerticaSource(Vertica_SQLAlchemySource):
```

Comment:
what's the reasoning behind this change?

Reply:
there were too many new codes for projections, ML Models, and Oauth-related metadata, which we didn't want to write in sql_common.py so it doesn't affect any other database. we created a common.py and did all our changes in that file to avoid code conflict.